Platform/CommonBoardPkg/CfgData/Template_BootOption.yaml

```diff
@@ -18,10 +18,12 @@ BOOT_OPTION_TMPL: >
     - ImageType_$(1) :
         name         : Image Type
         type         : Combo
-        option       : 0:Default, 1:Android, 2:ClearLinux, 3:Acrn, 4:Fastboot, 0xFE:Addendum, 0xFF:Not used
+        option       : 0:Default, 1:Android, 2:ClearLinux, 3:Acrn, 4:Fastboot, 0x1E:load and run extra image, 0x1F:PreOS default, 0xFF:Not used
         help         : >
                        Specify boot image type.
-                       Specially 'Addendum' indicates this option is not a standalone boot option. Instead, it provides additional information for the previous boot option.
+                       Image type 0x1E or 0x1F indicates this option is not a standalone boot option. Instead, it provides additional information for the previous boot option.
+                       0x1E means it need load and run extra image before normal OS image. The extra image will return to SBL and SBL will continue run normal OS.

```

Comment:
These values is not visisual in ConfigEditor. We should state the actual string in help.

Reply:
It is visual in Config editor. that's why we listed the options for preOS and extra image.